MapboxDirections/MBVisualInstructionType.swift

```diff
@@ -23,6 +23,16 @@ public enum VisualInstructionComponentType: Int, CustomStringConvertible {
      */
     case image
     
+    /**
+     The compoment contains the localized word for "exit".
+     */
+    case exit
+    
+    /**
+     A component contains an exit number.
+     */
+    case exitNumber
```

Comment:
Rename this property to `exitCode` for consistency with `RouteStep.exitCode`:

https://github.com/mapbox/MapboxDirections.swift/blob/1711560166601f46466391a21da1ff79f2727323/MapboxDirections/MBRouteStep.swift#L882-L889

This property was so named to avoid confusion with a preexisting `exitIndex` and also because a freeway exit number may be alphabetic or alphanumeric.